pkg/volume/rbd/rbd_util.go

```diff
@@ -233,13 +352,31 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) (string, error) {
 		}
 	}
 
-	devicePath, found := waitForPath(b.Pool, b.Image, 1)
-	if !found {
-		_, err = b.exec.Run("modprobe", "rbd")
-		if err != nil {
-			glog.Warningf("rbd: failed to load rbd kernel module:%v", err)
-		}
+	// If rbd-nbd tools are installed, we will try to map as a nbd device first.
+	nbdToolsFound := false
+	_, err = b.exec.Run("modprobe", "nbd")
+	if err != nil {
+		glog.V(5).Infof("rbd-nbd: nbd modprobe failed with error %v", err)
```

Comment:
should this log be of warning level? just like the one from `modprobe rbd`? Just to be consistent.

Reply:
rbd-nbd will be tried first, and we will fall back to rbd/krbd. Therefore, we've kept rbd/krbd warnings higher than rbd-nbd. We were afraid that existing users who leverage only rbd/krbd would be confused by a warning, when it isn't an issue for them.